test/org/impact2585/lib2585/tests/DrivetrainTest.java

```diff
@@ -0,0 +1,206 @@
+package org.impact2585.lib2585.tests;
+
+import org.impact2585.lib2585.Drivetrain;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import edu.wpi.first.wpilibj.RobotDrive;
+
+/**
+ * test for the wheel system
+ */
+public class DrivetrainTest {
+	private TestDrivetrain drivetrain;
+	private double driveForward;
+	private double currentRampForward;
+	private double rotate;
+	private boolean invert;
+	private boolean toggleRotationExponent;
+
+	/**
+	 * method to calculate the ramping
+	 * @return correct ramping
+	 */
+	public double rampForward() {
+		double realRampForward = (currentRampForward - driveForward) * drivetrain.getRamp()+ currentRampForward;
+		if (driveForward < drivetrain.getDeadzone() && driveForward > -drivetrain.getDeadzone())
+			realRampForward = 0;
+		return -realRampForward;
+	}
+
+	/**
+	 * Initializes the test wheel system and input
+	 */
+	@Before
+	public void setUp() {
+		drivetrain = new TestDrivetrain(.15, .6, 1, .5, null);
```

Comment:
Can you make the arguments being passed into the constructor constants? That way, it will be easier to see what each value is for if we need to change the unit test in the future.